connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectDistributed.java

```diff
@@ -106,6 +108,7 @@ public Connect startConnect(Map<String, String> workerProps) {
         // Create the admin client to be shared by all backing stores.
         Map<String, Object> adminProps = new HashMap<>(config.originals());
         ConnectUtils.addMetricsContextProperties(adminProps, config, kafkaClusterId);
+        adminProps.putIfAbsent(CLIENT_ID_CONFIG, config.groupId() + "-kafka-cluster-id-lookup");
```

Comment:
nit: this adminclient seems to do a lot more than looking up the kafka cluster id. It's also responsible for creating internal topics, verifying their settings, and retrieving end-offsets. perhaps `groupId-shared-admin`? `groupId-worker-internal`?

Reply:
Good catch; this is just a mistake. The admin client used for Kafka cluster ID lookup is instantiated elsewhere. I like `<groupId>-shared-admin`; will use that here.